<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartogramDialog</class>
 <widget class="QDialog" name="CartogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cartogram3</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="layerComboBoxLabel">
     <property name="text">
      <string>Input layer:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsMapLayerComboBox" name="layerComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="fieldListViewLabel">
     <property name="toolTip">
      <string>To compute cartograms for multiple fields, please use the Processing toolbox batch functionality.</string>
     </property>
     <property name="text">
      <string>Field:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsFieldListView" name="fieldListView">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="stopConditionsFrameLabel">
     <property name="text">
      <string>Stop conditions:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QFrame" name="stopConditionsFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="iterationsSpinBoxLabel">
        <property name="text">
         <string>max. number of iterations:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="iterationsSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="averageErrorDoubleSpinBoxLabel">
        <property name="text">
         <string>max. average error:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="averageErrorDoubleSpinBox">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="conditionsExplanationLabel">
        <property name="text">
         <string>Calculation stops as soon as one condition is met.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldListView</class>
   <extends>QListView</extends>
   <header>cartogram3.ui.qgsfieldlistview</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CartogramDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CartogramDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layerComboBox</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>fieldListView</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
